lib/testutils/minirunner/minirunner.go

```diff
@@ -39,7 +39,7 @@ var (
 // using a real JS runtime, it allows us to directly specify the options and
 // functions with Go code.
 type MiniRunner struct {
-	Fn              func(ctx context.Context, out chan<- stats.SampleContainer) error
+	Fn              func(ctx context.Context, state *lib.State, out chan<- stats.SampleContainer) error
```

Comment:
Hmm, I am starting to think that now, after we've refactored the JS module interface, we should also refactor `MiniRunner` to more closely resemble it  :thinking: For example, instead of passing a bunch of params, have a struct that resembles this? https://github.com/grafana/k6/blob/cd017d67db048ba85f5781dc50be13992698e666/js/modules/modules.go#L104-L117

@MStoykov, WDYT?

Reply:
Yeah, that will likely help. Although I don't see it using most of those, but still probably a good idea :+1:

Reply:
https://github.com/grafana/k6/issues/2382